drivers/entropy/entropy_stm32.c

```diff
@@ -176,7 +176,8 @@ static int entropy_stm32_rng_init(struct device *dev)
 	 *  Linear Feedback Shift Register
 	 */
 	 LL_RCC_SetRNGClockSource(LL_RCC_RNG_CLKSOURCE_PLLSAI1);
-#elif defined(RCC_HSI48_SUPPORT)
+#elif defined(RCC_CR2_HSI48ON) || defined(RCC_CR_HSI48ON) \
+	|| defined(RCC_CRRCR_HSI48ON)
```

Comment:
Can you split these changes in 2 commits? changes to entropy_stm32.c should be explained.

Reply:
Done